platform/device/builtin/db.go

```diff
@@ -46,6 +47,24 @@ func NewDB(db *bolt.DB) (*DB, error) {
 	return datastore, nil
 }
 
+func (db *DB) GetBootstrapToken(ctx context.Context, udid string) ([]byte, error) {
```

Comment:
nit: maybe return b64Data from the other definition?

```suggestion
func (db *DB) GetBootstrapToken(ctx context.Context, udid string) (B64Data, error) {
```